File: Java/Array/Count Odd and Even.java
//https://www.geeksforgeeks.org/problems/count-odd-even/1
class Solution {
    public int[] countOddEven(int[] arr) {
        // Code here
         int even=0,odd=0;
    for(int i=0;i<arr.length;i++)
    {
        if(arr[i]%2==0)
        {
        even++;
        }
        else
        {
            odd++;
        }
    }
    return new int[]{odd,even};
    }
}

File: Python/Array/Count Odd and Even.py
//https://www.geeksforgeeks.org/problems/count-odd-even/1
class Solution:
	def countOddEven(self, arr):
		#Code here
		odd = 0
        even = 0
        for num in arr:
            if num % 2 == 0:
                even += 1
            else:
                odd += 1
        return [odd, even]
